llvm/lib/MC/GOFFObjectWriter.cpp

```diff
@@ -295,6 +284,35 @@ uint64_t GOFFObjectWriter::writeObject(MCAssembler &Asm) {
   return OS.getWrittenSize();
 }
 
+namespace {
+
+class GOFFObjectWriter : public MCObjectWriter {
```

Comment:
If you will change AsmPrinter or a targetstreamer to be aware of GOFF-specifics, it would be better to expose the class to a header.

In 70c52b62c5669993e341664a63bfbe5245e32884 , I exported ELF and enabled a lot of simplification to MCAssembler/MCELFStreamer.

Reply:
That is a good point. I think we can also make use of a public available GOFFWriter. Only downside is that it pulls a couple of internal definitions into the header file, but that is the same as in the ELF implementation. Thanks for pointing that out.

Reply:
Ok, with the 2nd try it should now be correct....